config.py

```diff
@@ -1,7 +1,12 @@
+########################
+## RUNTIME VARIABLES ##
+########################
+ENVIRONMENT = "dev"  # "dev" or "prod"
```

Comment:
This should be an environment variable, since that's how every other volatile/secret thing is handled in this repository.

Reply:
added this in latest commit, I am sure there is a better way to implement it than what I have done.